src/index.html

```diff
@@ -5,8 +5,422 @@
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="Imac"
+        >
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="card__rating-stars card__rating-stars--5">
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+          </div>
+
+          <p class="card__rating-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="Imac"
+        >
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="card__rating-stars card__rating-stars--5">
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
+            <div class="card__rating-stars-star active"></div>
```

Comment:
Redundant, you can just remove it everywhere
```suggestion
            <div class="card__rating-stars-star"></div>
            <div class="card__rating-stars-star"></div>
            <div class="card__rating-stars-star"></div>
            <div class="card__rating-stars-star"></div>
            <div class="card__rating-stars-star"></div>
```

Reply:
Not fixed. Modifiers don't work 
<img width="503" alt="Screenshot 2023-11-19 at 16 02 46" src="https://github.com/mate-academy/layout_catalog/assets/135232001/c92cf64d-7c35-43d4-9cfb-faab275b120f">